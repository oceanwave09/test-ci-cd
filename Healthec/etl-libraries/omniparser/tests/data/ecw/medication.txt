apuid|RecordID|ItemID|PatientId|PatientAccountNo|EncounterId|ProviderId|ProviderNPI|MedicationOriginID|MedicationCodeSystem|MedicationCode|MedicationDescription|PrescriptionDate|PrescriptionBeginDate|PrescriptionEndDate|DaysSupply|Quantity|Refills|Strength|Formulation|Route|Frequency|Take|MedStatus
280807|280807.17502.2419588.12435990|266057|17502|17502|2419588|532670|1659584084|CM|NDC_CODE|00456-2020-01|Lexapro|2022-07-18|||90 days|90|0|20 mg|Tablet|Orally|daily|1 tablet|Taking
280807|280807.17502.2419588.12435991|266057|17502|17502|2419588|532670|1659584084|CM|NDC_CODE|50090-2279-00|Genvoya|2022-07-18||||||150-150-200-10 MG|Tablet|Orally|qd|as directed|Taking
280807|280807.17502.2504650.12683985|266057|17502|17502|2504650|543042|1245562859|TRMT|NDC_CODE|00074-6214-13|Depakote|2022-09-09|||30 day(s)|60 Tablet|1|250 MG|Tablet Delayed Release|Orally|bid|1 tab|Continue
280807|280807.17502.2504650.12683986|266057|17502|17502|2504650|543042|1245562859|TRMT|NDC_CODE|00074-6215-13|Depakote|2022-09-09|||30 day(s)|60 Tablet|1|500 MG|Tablet Delayed Release|Orally|bid|1 tab|Continue
280807|280807.17502.2504650.12683987|275788|17502|17502|2504650|543042|1245562859|TRMT|NDC_CODE|00310-0274-60|SEROquel|2022-09-09|||30 day(s)|30|1|300 MG|Tablet|Orally|qhs|1 tablet|Continue
280807|280807.17502.2504650.12683988|275788|17502|17502|2504650|543042|1245562859|TRMT|NDC_CODE|00310-0278-10|SEROquel|2022-09-09|||30 day(s)|90|1|50 MG|Tablet|Orally|as directed|1 tablet every morning, 2 tablets everyday at 3:00|Continue
280807|280807.17502.2528276.12706359|283418|17502|17502|2528276|532700|1780671446|CM|NDC_CODE|00032-3016-13|Creon|2022-09-14||||||36000-114000 UNIT|Capsule Delayed Release Particles|Orally|before breakfast, lunch, dinner and 1 caps before snack as needed|2 cap(s)|Taking
280807|280807.17502.2528276.12706360|277438|17502|17502|2528276|532700|1780671446|CM|NDC_CODE|11917-04865|Tolnaftate|2022-09-14||||||1 %|Aerosol Powder|Externally to feet|Twice a day|1 application|Taking
280807|280807.17502.2528276.12706361|261712|17502|17502|2528276|532700|1780671446|CM|NDC_CODE|00440-7010-01|Acetaminophen|2022-09-14||||||325 MG|Tablet|Orally|every 6 hrs|2 tablets as needed|Taking
280807|280807.17502.2528276.12706362|275185|17502|17502|2528276|532700|1780671446|CM|NDC_CODE|00023-9163-30|Restasis|2022-09-14||||||0.05 %|Emulsion|Ophthalmic|Twice a day|1 drop in each eye|Taking
280807|280807.17502.2528276.12706363|338273|17502|17502|2528276|532700|1780671446|CM|NDC_CODE|80681-02000|Multivitamin|2022-09-14|||30 day(s)|30||-|Tablet|Orally|Once a day|1 tablet|Taking
280807|280807.17502.2528276.12706364|276998|17502|17502|2528276|532700|1780671446|CM|NDC_CODE|00228-2996-11|Tamsulosin HCl|2022-09-14|||30 day(s)|30||0.4 MG|Capsule|Orally|Once a day|1 capsule|Taking
280807|280807.17502.2528276.12706365|337918|17502|17502|2528276|532700|1780671446|CM|NDC_CODE|50458-0140-30|Invokana|2022-09-14|||30 day(s)|30||100 MG|Tablet|Orally|Once a day|1 tablet before the first meal of the day|Taking
280807|280807.17502.2528276.12706366|270430|17502|17502|2528276|532700|1780671446|CM|NDC_CODE|00185-0610-01|Lisinopril|2022-09-14|||30 day(s)|30||10 MG|Tablet|Orally|Once a day|1 tablet|Taking
280807|280807.17502.2528276.12706367|272590|17502|17502|2528276|532700|1780671446|CM|NDC_CODE|00781-2234-01|Omeprazole|2022-09-14|||30||0|40 MG|Capsule Delayed Release|orally|bid|1 capsule|Taking
280807|280807.17502.2528276.12706368|272613|17502|17502|2528276|532700|1780671446|CM|NDC_CODE|37205008082|One Daily|2022-09-14|||30|30 Unspecified|2|-|Tablet|Oral||TAKE 1 TABLET BY MOUTH EVERY DAY|Taking
280807|280807.17502.2528276.12706369|338181|17502|17502|2528276|532700|1780671446|CM|NDC_CODE|00378-4598-05|Metoprolol Succinate ER|2022-09-14||||||200 MG|Tablet Extended Release 24 Hour|Orally|Once a day|TAKE 1 TABLET (50MG)  BY  MOUTH  EVERY DAY|Taking
280807|280807.17502.2528276.12706370|271165|17502|17502|2528276|532700|1780671446|CM|NDC_CODE|00378-7185-05|metFORMIN HCl|2022-09-14||||||500 MG|Tablet|Orally|BID|1 tablet with a meal|Taking
280807|280807.17502.2528276.12706371|268425|17502|17502|2528276|532700|1780671446|CM|NDC_CODE|00440-6565-01|Glimepiride|2022-09-14||||||2 MG|Tablet|PO|BID|1 tablet|Taking
280807|280807.17502.2528276.12706372|269017|17502|17502|2528276|532700|1780671446|CM|NDC_CODE|00378-0327-01|Haloperidol|2022-09-14|||30 days|45|0|5 MG|Tablet|Orally|bid|1/2 QAM ,1QPM|Taking
280807|280807.17502.2528276.12706373|274580|17502|17502|2528276|532700|1780671446|CM|NDC_CODE|00093-8162-01|QUEtiapine Fumarate|2022-09-14|||30 day(s)|30|3|100 MG|Tablet|Orally|Once a day|1 tablet at bedtime|Taking
280807|280807.17502.2528276.12706374|271071|17502|17502|2528276|532700|1780671446|CM|NDC_CODE|07610015540|Melatonin|2022-09-14|||30 days|60|3|5 MG|Tablet|Oral|qhs|TAKE 2 TABLET AT BEDTIME BY MOUTH|Taking
280807|280807.17502.2528276.12706375|266058|17502|17502|2528276|532700|1780671446|CM|NDC_CODE|00074-3826-13|Depakote ER|2022-09-14|2021-11-04||30 day(s)|60|2|250 MG|Tablet Extended Release 24 Hour|Orally|2 QAM|as directed|Taking
280807|280807.17502.2528276.12706376|268965|17502|17502|2528276|532700|1780671446|CM|NDC_CODE|00591-0444-01|guanFACINE HCl|2022-09-14|||30 day(s)|60 Tablet|3|1 MG|Tablet|Orally|BID|1TAB|Taking
280807|280807.17502.2528276.12706378|277631|17502|17502|2528276|532700|1780671446|CM|NDC_CODE|00591-5335-01|Trihexyphenidyl HCl|2022-09-14|||30 days|60|3|2 MG|Tablet|Orally|bid|1 TABS|Taking
280807|280807.17502.2528276.12706379|263398|17502|17502|2528276|532700|1780671446|CM|NDC_CODE|00603243732|Benztropine Mesylate|2022-09-14|||30 day(s)|90|3|0.5 MG|Tablet|Orally|1 in am ,2 hs|1 tablet|Taking
280807|280807.17502.2528276.12706380|270590|17502|17502|2528276|532700|1780671446|CM|NDC_CODE|00378-2457-01|LORazepam|2022-09-14|2022-09-13||30 days|30|0|1 MG|Tablet|Orally|QHS|1 tab(s)|Taking
280807|280807.17502.2528276.12706381|270590|17502|17502|2528276|532700|1780671446|CM|NDC_CODE|00591-0240-01|LORazepam|2022-09-14|2022-09-13||30 days|30|0|0.5 MG|Tablet|Orally|qam|1 tablet|Taking
280807|280807.17502.2528276.12706382|262894|17502|17502|2528276|532700|1780671446|CM|NDC_CODE|00363041474|Aspirin|2022-09-14|||30 days|30 Tablet|5|81 MG|Tablet Delayed Release|Oral|Once a day|1 tablet|Unknown
