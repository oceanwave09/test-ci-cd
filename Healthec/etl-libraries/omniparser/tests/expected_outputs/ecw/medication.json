[
	{
		"apuid": "280807",
		"days_supply": "90 days",
		"encounter_id": "2419588",
		"formulation": "Tablet",
		"frequency": "daily",
		"item_id": "266057",
		"med_status": "Taking",
		"medication_code": "00456-2020-01",
		"medication_code_system": "NDC_CODE",
		"medication_description": "Lexapro",
		"medication_origin_id": "CM",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_date": "2022-07-18",
		"provider_id": "532670",
		"provider_npi": "67eaba81cfd50419ca1068ac5ce03825",
		"quantity": "90",
		"record_id": "280807.17502.2419588.12435990",
		"refills": "0",
		"route": "Orally",
		"row_id": "3b60aa7e-f66e-4cb4-b14b-e335df8f193e",
		"strength": "20 mg",
		"take": "1 tablet"
	},
	{
		"apuid": "280807",
		"encounter_id": "2419588",
		"formulation": "Tablet",
		"frequency": "qd",
		"item_id": "266057",
		"med_status": "Taking",
		"medication_code": "50090-2279-00",
		"medication_code_system": "NDC_CODE",
		"medication_description": "Genvoya",
		"medication_origin_id": "CM",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_date": "2022-07-18",
		"provider_id": "532670",
		"provider_npi": "67eaba81cfd50419ca1068ac5ce03825",
		"record_id": "280807.17502.2419588.12435991",
		"route": "Orally",
		"row_id": "192cc92b-e50e-408a-a296-1ab6a6cdace4",
		"strength": "150-150-200-10 MG",
		"take": "as directed"
	},
	{
		"apuid": "280807",
		"days_supply": "30 day(s)",
		"encounter_id": "2504650",
		"formulation": "Tablet Delayed Release",
		"frequency": "bid",
		"item_id": "266057",
		"med_status": "Continue",
		"medication_code": "00074-6214-13",
		"medication_code_system": "NDC_CODE",
		"medication_description": "Depakote",
		"medication_origin_id": "TRMT",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_date": "2022-09-09",
		"provider_id": "543042",
		"provider_npi": "e6cedade0238409d33dbac800d6d7fc1",
		"quantity": "60 Tablet",
		"record_id": "280807.17502.2504650.12683985",
		"refills": "1",
		"route": "Orally",
		"row_id": "d47a4b27-c9ba-4fcb-9775-1c8120e2e186",
		"strength": "250 MG",
		"take": "1 tab"
	},
	{
		"apuid": "280807",
		"days_supply": "30 day(s)",
		"encounter_id": "2504650",
		"formulation": "Tablet Delayed Release",
		"frequency": "bid",
		"item_id": "266057",
		"med_status": "Continue",
		"medication_code": "00074-6215-13",
		"medication_code_system": "NDC_CODE",
		"medication_description": "Depakote",
		"medication_origin_id": "TRMT",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_date": "2022-09-09",
		"provider_id": "543042",
		"provider_npi": "e6cedade0238409d33dbac800d6d7fc1",
		"quantity": "60 Tablet",
		"record_id": "280807.17502.2504650.12683986",
		"refills": "1",
		"route": "Orally",
		"row_id": "d9f8e788-a664-4d6b-941c-285205ef56e2",
		"strength": "500 MG",
		"take": "1 tab"
	},
	{
		"apuid": "280807",
		"days_supply": "30 day(s)",
		"encounter_id": "2504650",
		"formulation": "Tablet",
		"frequency": "qhs",
		"item_id": "275788",
		"med_status": "Continue",
		"medication_code": "00310-0274-60",
		"medication_code_system": "NDC_CODE",
		"medication_description": "SEROquel",
		"medication_origin_id": "TRMT",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_date": "2022-09-09",
		"provider_id": "543042",
		"provider_npi": "e6cedade0238409d33dbac800d6d7fc1",
		"quantity": "30",
		"record_id": "280807.17502.2504650.12683987",
		"refills": "1",
		"route": "Orally",
		"row_id": "e6dd95cb-6687-4d69-a786-ca0e185ee999",
		"strength": "300 MG",
		"take": "1 tablet"
	},
	{
		"apuid": "280807",
		"days_supply": "30 day(s)",
		"encounter_id": "2504650",
		"formulation": "Tablet",
		"frequency": "as directed",
		"item_id": "275788",
		"med_status": "Continue",
		"medication_code": "00310-0278-10",
		"medication_code_system": "NDC_CODE",
		"medication_description": "SEROquel",
		"medication_origin_id": "TRMT",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_date": "2022-09-09",
		"provider_id": "543042",
		"provider_npi": "e6cedade0238409d33dbac800d6d7fc1",
		"quantity": "90",
		"record_id": "280807.17502.2504650.12683988",
		"refills": "1",
		"route": "Orally",
		"row_id": "a2820488-efc7-4051-b141-e5f4284eb22a",
		"strength": "50 MG",
		"take": "1 tablet every morning, 2 tablets everyday at 3:00"
	},
	{
		"apuid": "280807",
		"encounter_id": "2528276",
		"formulation": "Capsule Delayed Release Particles",
		"frequency": "before breakfast, lunch, dinner and 1 caps before snack as needed",
		"item_id": "283418",
		"med_status": "Taking",
		"medication_code": "00032-3016-13",
		"medication_code_system": "NDC_CODE",
		"medication_description": "Creon",
		"medication_origin_id": "CM",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_date": "2022-09-14",
		"provider_id": "532700",
		"provider_npi": "6d1c6795aff803d71fdb11dd1b7594d2",
		"record_id": "280807.17502.2528276.12706359",
		"route": "Orally",
		"row_id": "f3b1dd2f-0188-48ae-a61e-79182f558b16",
		"strength": "36000-114000 UNIT",
		"take": "2 cap(s)"
	},
	{
		"apuid": "280807",
		"encounter_id": "2528276",
		"formulation": "Aerosol Powder",
		"frequency": "Twice a day",
		"item_id": "277438",
		"med_status": "Taking",
		"medication_code": "11917-04865",
		"medication_code_system": "NDC_CODE",
		"medication_description": "Tolnaftate",
		"medication_origin_id": "CM",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_date": "2022-09-14",
		"provider_id": "532700",
		"provider_npi": "6d1c6795aff803d71fdb11dd1b7594d2",
		"record_id": "280807.17502.2528276.12706360",
		"route": "Externally to feet",
		"row_id": "4db0e591-2b67-4870-903f-67ce311c9bfa",
		"strength": "1 %",
		"take": "1 application"
	},
	{
		"apuid": "280807",
		"encounter_id": "2528276",
		"formulation": "Tablet",
		"frequency": "every 6 hrs",
		"item_id": "261712",
		"med_status": "Taking",
		"medication_code": "00440-7010-01",
		"medication_code_system": "NDC_CODE",
		"medication_description": "Acetaminophen",
		"medication_origin_id": "CM",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_date": "2022-09-14",
		"provider_id": "532700",
		"provider_npi": "6d1c6795aff803d71fdb11dd1b7594d2",
		"record_id": "280807.17502.2528276.12706361",
		"route": "Orally",
		"row_id": "5a36632d-9a24-41e6-9090-8aed69302bc3",
		"strength": "325 MG",
		"take": "2 tablets as needed"
	},
	{
		"apuid": "280807",
		"encounter_id": "2528276",
		"formulation": "Emulsion",
		"frequency": "Twice a day",
		"item_id": "275185",
		"med_status": "Taking",
		"medication_code": "00023-9163-30",
		"medication_code_system": "NDC_CODE",
		"medication_description": "Restasis",
		"medication_origin_id": "CM",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_date": "2022-09-14",
		"provider_id": "532700",
		"provider_npi": "6d1c6795aff803d71fdb11dd1b7594d2",
		"record_id": "280807.17502.2528276.12706362",
		"route": "Ophthalmic",
		"row_id": "089f098b-105b-4027-9209-ae1e98631909",
		"strength": "0.05 %",
		"take": "1 drop in each eye"
	},
	{
		"apuid": "280807",
		"days_supply": "30 day(s)",
		"encounter_id": "2528276",
		"formulation": "Tablet",
		"frequency": "Once a day",
		"item_id": "338273",
		"med_status": "Taking",
		"medication_code": "80681-02000",
		"medication_code_system": "NDC_CODE",
		"medication_description": "Multivitamin",
		"medication_origin_id": "CM",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_date": "2022-09-14",
		"provider_id": "532700",
		"provider_npi": "6d1c6795aff803d71fdb11dd1b7594d2",
		"quantity": "30",
		"record_id": "280807.17502.2528276.12706363",
		"route": "Orally",
		"row_id": "8de8320f-8617-4ebe-ac1c-4da6a02a56e3",
		"strength": "-",
		"take": "1 tablet"
	},
	{
		"apuid": "280807",
		"days_supply": "30 day(s)",
		"encounter_id": "2528276",
		"formulation": "Capsule",
		"frequency": "Once a day",
		"item_id": "276998",
		"med_status": "Taking",
		"medication_code": "00228-2996-11",
		"medication_code_system": "NDC_CODE",
		"medication_description": "Tamsulosin HCl",
		"medication_origin_id": "CM",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_date": "2022-09-14",
		"provider_id": "532700",
		"provider_npi": "6d1c6795aff803d71fdb11dd1b7594d2",
		"quantity": "30",
		"record_id": "280807.17502.2528276.12706364",
		"route": "Orally",
		"row_id": "9a754caa-3c7e-48a0-aedf-e15f275d51d7",
		"strength": "0.4 MG",
		"take": "1 capsule"
	},
	{
		"apuid": "280807",
		"days_supply": "30 day(s)",
		"encounter_id": "2528276",
		"formulation": "Tablet",
		"frequency": "Once a day",
		"item_id": "337918",
		"med_status": "Taking",
		"medication_code": "50458-0140-30",
		"medication_code_system": "NDC_CODE",
		"medication_description": "Invokana",
		"medication_origin_id": "CM",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_date": "2022-09-14",
		"provider_id": "532700",
		"provider_npi": "6d1c6795aff803d71fdb11dd1b7594d2",
		"quantity": "30",
		"record_id": "280807.17502.2528276.12706365",
		"route": "Orally",
		"row_id": "086f114b-7d4f-40a7-bf2e-c519659d15ba",
		"strength": "100 MG",
		"take": "1 tablet before the first meal of the day"
	},
	{
		"apuid": "280807",
		"days_supply": "30 day(s)",
		"encounter_id": "2528276",
		"formulation": "Tablet",
		"frequency": "Once a day",
		"item_id": "270430",
		"med_status": "Taking",
		"medication_code": "00185-0610-01",
		"medication_code_system": "NDC_CODE",
		"medication_description": "Lisinopril",
		"medication_origin_id": "CM",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_date": "2022-09-14",
		"provider_id": "532700",
		"provider_npi": "6d1c6795aff803d71fdb11dd1b7594d2",
		"quantity": "30",
		"record_id": "280807.17502.2528276.12706366",
		"route": "Orally",
		"row_id": "768e13a2-2194-4da9-b5cd-14b3b5661d27",
		"strength": "10 MG",
		"take": "1 tablet"
	},
	{
		"apuid": "280807",
		"days_supply": "30",
		"encounter_id": "2528276",
		"formulation": "Capsule Delayed Release",
		"frequency": "bid",
		"item_id": "272590",
		"med_status": "Taking",
		"medication_code": "00781-2234-01",
		"medication_code_system": "NDC_CODE",
		"medication_description": "Omeprazole",
		"medication_origin_id": "CM",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_date": "2022-09-14",
		"provider_id": "532700",
		"provider_npi": "6d1c6795aff803d71fdb11dd1b7594d2",
		"record_id": "280807.17502.2528276.12706367",
		"refills": "0",
		"route": "orally",
		"row_id": "c283cbeb-eb1c-4295-bb3c-de171c27952a",
		"strength": "40 MG",
		"take": "1 capsule"
	},
	{
		"apuid": "280807",
		"days_supply": "30",
		"encounter_id": "2528276",
		"formulation": "Tablet",
		"item_id": "272613",
		"med_status": "Taking",
		"medication_code": "37205008082",
		"medication_code_system": "NDC_CODE",
		"medication_description": "One Daily",
		"medication_origin_id": "CM",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_date": "2022-09-14",
		"provider_id": "532700",
		"provider_npi": "6d1c6795aff803d71fdb11dd1b7594d2",
		"quantity": "30 Unspecified",
		"record_id": "280807.17502.2528276.12706368",
		"refills": "2",
		"route": "Oral",
		"row_id": "49de884a-ca6a-4b48-9afa-fd9b562ee266",
		"strength": "-",
		"take": "TAKE 1 TABLET BY MOUTH EVERY DAY"
	},
	{
		"apuid": "280807",
		"encounter_id": "2528276",
		"formulation": "Tablet Extended Release 24 Hour",
		"frequency": "Once a day",
		"item_id": "338181",
		"med_status": "Taking",
		"medication_code": "00378-4598-05",
		"medication_code_system": "NDC_CODE",
		"medication_description": "Metoprolol Succinate ER",
		"medication_origin_id": "CM",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_date": "2022-09-14",
		"provider_id": "532700",
		"provider_npi": "6d1c6795aff803d71fdb11dd1b7594d2",
		"record_id": "280807.17502.2528276.12706369",
		"route": "Orally",
		"row_id": "d91a3ffe-ea52-4b30-a8dc-e290be485fa8",
		"strength": "200 MG",
		"take": "TAKE 1 TABLET (50MG)  BY  MOUTH  EVERY DAY"
	},
	{
		"apuid": "280807",
		"encounter_id": "2528276",
		"formulation": "Tablet",
		"frequency": "BID",
		"item_id": "271165",
		"med_status": "Taking",
		"medication_code": "00378-7185-05",
		"medication_code_system": "NDC_CODE",
		"medication_description": "metFORMIN HCl",
		"medication_origin_id": "CM",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_date": "2022-09-14",
		"provider_id": "532700",
		"provider_npi": "6d1c6795aff803d71fdb11dd1b7594d2",
		"record_id": "280807.17502.2528276.12706370",
		"route": "Orally",
		"row_id": "78f2b97d-7065-4e61-a9cb-5ae8b368c391",
		"strength": "500 MG",
		"take": "1 tablet with a meal"
	},
	{
		"apuid": "280807",
		"encounter_id": "2528276",
		"formulation": "Tablet",
		"frequency": "BID",
		"item_id": "268425",
		"med_status": "Taking",
		"medication_code": "00440-6565-01",
		"medication_code_system": "NDC_CODE",
		"medication_description": "Glimepiride",
		"medication_origin_id": "CM",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_date": "2022-09-14",
		"provider_id": "532700",
		"provider_npi": "6d1c6795aff803d71fdb11dd1b7594d2",
		"record_id": "280807.17502.2528276.12706371",
		"route": "PO",
		"row_id": "4e6fb486-194e-4a86-92e7-082f66fd8936",
		"strength": "2 MG",
		"take": "1 tablet"
	},
	{
		"apuid": "280807",
		"days_supply": "30 days",
		"encounter_id": "2528276",
		"formulation": "Tablet",
		"frequency": "bid",
		"item_id": "269017",
		"med_status": "Taking",
		"medication_code": "00378-0327-01",
		"medication_code_system": "NDC_CODE",
		"medication_description": "Haloperidol",
		"medication_origin_id": "CM",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_date": "2022-09-14",
		"provider_id": "532700",
		"provider_npi": "6d1c6795aff803d71fdb11dd1b7594d2",
		"quantity": "45",
		"record_id": "280807.17502.2528276.12706372",
		"refills": "0",
		"route": "Orally",
		"row_id": "f1134684-93b1-4146-b1de-2ffaa1741ba4",
		"strength": "5 MG",
		"take": "1/2 QAM ,1QPM"
	},
	{
		"apuid": "280807",
		"days_supply": "30 day(s)",
		"encounter_id": "2528276",
		"formulation": "Tablet",
		"frequency": "Once a day",
		"item_id": "274580",
		"med_status": "Taking",
		"medication_code": "00093-8162-01",
		"medication_code_system": "NDC_CODE",
		"medication_description": "QUEtiapine Fumarate",
		"medication_origin_id": "CM",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_date": "2022-09-14",
		"provider_id": "532700",
		"provider_npi": "6d1c6795aff803d71fdb11dd1b7594d2",
		"quantity": "30",
		"record_id": "280807.17502.2528276.12706373",
		"refills": "3",
		"route": "Orally",
		"row_id": "665ca451-1c69-461d-9c4a-336f2e54a22f",
		"strength": "100 MG",
		"take": "1 tablet at bedtime"
	},
	{
		"apuid": "280807",
		"days_supply": "30 days",
		"encounter_id": "2528276",
		"formulation": "Tablet",
		"frequency": "qhs",
		"item_id": "271071",
		"med_status": "Taking",
		"medication_code": "07610015540",
		"medication_code_system": "NDC_CODE",
		"medication_description": "Melatonin",
		"medication_origin_id": "CM",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_date": "2022-09-14",
		"provider_id": "532700",
		"provider_npi": "6d1c6795aff803d71fdb11dd1b7594d2",
		"quantity": "60",
		"record_id": "280807.17502.2528276.12706374",
		"refills": "3",
		"route": "Oral",
		"row_id": "f924851a-ecde-4ca1-8fe0-a24ce3f450e7",
		"strength": "5 MG",
		"take": "TAKE 2 TABLET AT BEDTIME BY MOUTH"
	},
	{
		"apuid": "280807",
		"days_supply": "30 day(s)",
		"encounter_id": "2528276",
		"formulation": "Tablet Extended Release 24 Hour",
		"frequency": "2 QAM",
		"item_id": "266058",
		"med_status": "Taking",
		"medication_code": "00074-3826-13",
		"medication_code_system": "NDC_CODE",
		"medication_description": "Depakote ER",
		"medication_origin_id": "CM",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_begin_date": "2021-11-04",
		"prescription_date": "2022-09-14",
		"provider_id": "532700",
		"provider_npi": "6d1c6795aff803d71fdb11dd1b7594d2",
		"quantity": "60",
		"record_id": "280807.17502.2528276.12706375",
		"refills": "2",
		"route": "Orally",
		"row_id": "7d96a614-1f4b-45c0-83a2-076e4fcbbe71",
		"strength": "250 MG",
		"take": "as directed"
	},
	{
		"apuid": "280807",
		"days_supply": "30 day(s)",
		"encounter_id": "2528276",
		"formulation": "Tablet",
		"frequency": "BID",
		"item_id": "268965",
		"med_status": "Taking",
		"medication_code": "00591-0444-01",
		"medication_code_system": "NDC_CODE",
		"medication_description": "guanFACINE HCl",
		"medication_origin_id": "CM",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_date": "2022-09-14",
		"provider_id": "532700",
		"provider_npi": "6d1c6795aff803d71fdb11dd1b7594d2",
		"quantity": "60 Tablet",
		"record_id": "280807.17502.2528276.12706376",
		"refills": "3",
		"route": "Orally",
		"row_id": "0007cffe-362b-4b7c-9f58-5696ffe94691",
		"strength": "1 MG",
		"take": "1TAB"
	},
	{
		"apuid": "280807",
		"days_supply": "30 days",
		"encounter_id": "2528276",
		"formulation": "Tablet",
		"frequency": "bid",
		"item_id": "277631",
		"med_status": "Taking",
		"medication_code": "00591-5335-01",
		"medication_code_system": "NDC_CODE",
		"medication_description": "Trihexyphenidyl HCl",
		"medication_origin_id": "CM",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_date": "2022-09-14",
		"provider_id": "532700",
		"provider_npi": "6d1c6795aff803d71fdb11dd1b7594d2",
		"quantity": "60",
		"record_id": "280807.17502.2528276.12706378",
		"refills": "3",
		"route": "Orally",
		"row_id": "13d23073-5d46-472d-9de7-e2a96cf8ca12",
		"strength": "2 MG",
		"take": "1 TABS"
	},
	{
		"apuid": "280807",
		"days_supply": "30 day(s)",
		"encounter_id": "2528276",
		"formulation": "Tablet",
		"frequency": "1 in am ,2 hs",
		"item_id": "263398",
		"med_status": "Taking",
		"medication_code": "00603243732",
		"medication_code_system": "NDC_CODE",
		"medication_description": "Benztropine Mesylate",
		"medication_origin_id": "CM",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_date": "2022-09-14",
		"provider_id": "532700",
		"provider_npi": "6d1c6795aff803d71fdb11dd1b7594d2",
		"quantity": "90",
		"record_id": "280807.17502.2528276.12706379",
		"refills": "3",
		"route": "Orally",
		"row_id": "4f74f4cf-6500-40e4-9484-bffdfd4db189",
		"strength": "0.5 MG",
		"take": "1 tablet"
	},
	{
		"apuid": "280807",
		"days_supply": "30 days",
		"encounter_id": "2528276",
		"formulation": "Tablet",
		"frequency": "QHS",
		"item_id": "270590",
		"med_status": "Taking",
		"medication_code": "00378-2457-01",
		"medication_code_system": "NDC_CODE",
		"medication_description": "LORazepam",
		"medication_origin_id": "CM",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_begin_date": "2022-09-13",
		"prescription_date": "2022-09-14",
		"provider_id": "532700",
		"provider_npi": "6d1c6795aff803d71fdb11dd1b7594d2",
		"quantity": "30",
		"record_id": "280807.17502.2528276.12706380",
		"refills": "0",
		"route": "Orally",
		"row_id": "809cd6db-d774-4891-bfa0-255866e6ec63",
		"strength": "1 MG",
		"take": "1 tab(s)"
	},
	{
		"apuid": "280807",
		"days_supply": "30 days",
		"encounter_id": "2528276",
		"formulation": "Tablet",
		"frequency": "qam",
		"item_id": "270590",
		"med_status": "Taking",
		"medication_code": "00591-0240-01",
		"medication_code_system": "NDC_CODE",
		"medication_description": "LORazepam",
		"medication_origin_id": "CM",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_begin_date": "2022-09-13",
		"prescription_date": "2022-09-14",
		"provider_id": "532700",
		"provider_npi": "6d1c6795aff803d71fdb11dd1b7594d2",
		"quantity": "30",
		"record_id": "280807.17502.2528276.12706381",
		"refills": "0",
		"route": "Orally",
		"row_id": "d7b22fbf-c114-4d1d-b1f1-8ddcb139932e",
		"strength": "0.5 MG",
		"take": "1 tablet"
	},
	{
		"apuid": "280807",
		"days_supply": "30 days",
		"encounter_id": "2528276",
		"formulation": "Tablet Delayed Release",
		"frequency": "Once a day",
		"item_id": "262894",
		"med_status": "Unknown",
		"medication_code": "00363041474",
		"medication_code_system": "NDC_CODE",
		"medication_description": "Aspirin",
		"medication_origin_id": "CM",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"prescription_date": "2022-09-14",
		"provider_id": "532700",
		"provider_npi": "6d1c6795aff803d71fdb11dd1b7594d2",
		"quantity": "30 Tablet",
		"record_id": "280807.17502.2528276.12706382",
		"refills": "5",
		"route": "Oral",
		"row_id": "7aaea82b-28d5-4102-be34-93e7ce98fd42",
		"strength": "81 MG",
		"take": "1 tablet"
	}
]
